### PR TITLE
MAT-7422,7440 added function and fluent function options in cql builder

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADiE PR

Jira Ticket: [MAT-7422](https://jira.cms.gov/browse/MAT-7422),[MAT-7440](https://jira.cms.gov/browse/MAT-7440)
(Optional) Related Tickets:

### Summary
added function and fluent function options in cql builder panel

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is in to the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/CqlBuilderPanel/CqlBuilderPanel.tsx [hunk 1]
```diff
@@ -7,7 +7,10 @@ import { useFeatureFlags } from "@madie/madie-util";
 import IncludesTabSection from "./Includes/Includes";
 import useQdmElmTranslationServiceApi from "../api/useQdmElmTranslationServiceApi";
 import useFhirElmTranslationServiceApi from "../api/useFhirElmTranslationServiceApi";
-import { CqlBuilderLookup } from "../model/CqlBuilderLookup";
+import {
+  CqlBuilderAvailableLookupData,
```
Comment (adongare) [inline src/CqlBuilderPanel/CqlBuilderPanel.tsx]: Not sure what is AvailableLookupData. will there be UnavailableData as well?

**Review by adongare — DISMISSED** [review]

File: src/CqlBuilderPanel/CqlBuilderPanel.tsx (hunk 1 above)
Comment (sb-prateekkeerthi) [reply src/CqlBuilderPanel/CqlBuilderPanel.tsx]: should I change it to CqlBuilderLookupData

Comment (adongare) [reply src/CqlBuilderPanel/CqlBuilderPanel.tsx]: Yeah, that looks more appropriate.

**Review by adongare — APPROVED** [review]
Looks good!

**Pull request merged**